assets/src/components/PostMeta/withPostMeta.js

```diff
@@ -38,9 +38,11 @@ export function withPostMeta( WrappedComponent ) {
       };
     }
 
-    handleChange( metaKey, value ) {
-      const getNewMeta = this.props.getNewMeta || (( metaKey, value, meta ) => ({ [ metaKey ]: value }));
-      const meta = getNewMeta(metaKey, value, this.props.postMeta );
+    async handleChange( metaKey, value ) {
+      const getNewMeta = this.props.getNewMeta || (( metaKey, value, meta ) => {
```

Comment:
I found it a bit hard to guess what's happening with `getNewMeta`,  it seems that there are a few flavors of it: this "default" definition if the prop is not set, then there is a `provideNewMeta` def in `fromThemeOptions`, and it also seems to be overriden by `handleThemeSwitch` in the `ThemeSelect` compo, I think it makes it a bit hard to see which version the component is actually going to get at the end.